### PR TITLE
Implement DEV-218 & DEV-219 and commit a starting point for the Emote app

### PR DESCRIPTION
**Pull request by zfox23**
The emote app is disabled in the script by default.

[DEV-218](https://highfidelity.atlassian.net/browse/DEV-218)
[DEV-219](https://highfidelity.atlassian.net/browse/DEV-219)

I am wondering if we should implement a different InteractiveWindow type like `CUSTOM` that does not contain any QML wrapper around the source like the regular InteractiveWindow does (with `InteractiveWindow.qml`). That would prevent things like the strange white artifact appearing in dark borderless QML windows. Feel free to talk to me about this if you see this message and want to know more.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20276/HighFidelity-Beta-PR15897-d622e53.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20276/HighFidelity-Beta-Interface-PR15897-d622e53.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20276/HighFidelity-Beta-PR15897-d622e53.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20276/HighFidelity-Quest-Beta-PR15897-d622e53.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20276/HighFidelity-Beta-PR15897-d622e53.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20276/HighFidelity-Beta-Interface-PR15897-d622e53.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20294/HighFidelity-Beta-PR15897-4dfbc48.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20294/HighFidelity-Quest-Beta-PR15897-4dfbc48.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20294/HighFidelity-Beta-PR15897-4dfbc48.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20294/HighFidelity-Beta-Interface-PR15897-4dfbc48.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20294/HighFidelity-Beta-PR15897-4dfbc48.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20294/HighFidelity-Beta-Interface-PR15897-4dfbc48.exe

File: interface/resources/qml/hifi/simplifiedUI/emoteApp/bar/EmoteAppBar.qml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+//  Created by Zach Fox on 2019-07-08
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import "../../simplifiedConstants" as SimplifiedConstants
```
Comment (RebeccaStankus) [inline interface/resources/qml/hifi/simplifiedUI/emoteApp/bar/EmoteAppBar.qml]: Can these be added to the qml directory and be imported as modules?

Comment (zfox23) [reply interface/resources/qml/hifi/simplifiedUI/emoteApp/bar/EmoteAppBar.qml]: Great question, resolved on Slack.

**Pull request merged**